pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/api.go

```diff
@@ -172,6 +172,16 @@ func (s *apiScaffolder) scafffoldControllerWithImage(scaffold *machinery.Scaffol
 			return fmt.Errorf("error scaffolding container port in the controller: %v", err)
 		}
 	}
+
+	runAsUser := "1000"
+	if len(s.runAsUser) > 0 {
+		runAsUser = s.runAsUser
+	}
```

Comment:
This suggestion was before because the initial idea was:
- if flag not informed does not scaffold runAsUser
- If flag informed then scaffold runAsUser with default user (only --run-as-user without value)
- if flag informed with an value then scaffold using the value informed.

However, I am also ok with we keep that simple by:
- if the flag is informed then scaffold with the value informed 
- if flag NOT informed then does not scaffold runAsUser